src/ray/util/filesystem.h

```diff
@@ -22,68 +23,24 @@
 
 namespace ray {
 
-/// \return The portable directory separator (slash on all OSes).
-static inline char GetAltDirSep() { return '/'; }
-
 /// \return The platform directory separator (backslash on Windows, slash on other OSes).
-static inline char GetDirSep() {
-  char result;
-#ifdef _WIN32
-  result = '\\';
-#else
-  result = '/';
-#endif
-  return result;
-}
-
-/// \return The platform PATH separator (semicolon on Windows, colon on other OSes).
-static inline char GetPathSep() {
-  char result;
-#ifdef _WIN32
-  result = ';';
-#else
-  result = ':';
-#endif
-  return result;
-}
-
-/// Returns the executable binary suffix for the platform, if any.
-std::string GetExeSuffix();
+static inline char GetDirSep() { return std::filesystem::path::preferred_separator; }
```

Comment:
I think I would just remove this one and change callers to use `std::filesystem::path::preferred_separator` directly.

Reply:
removed.